omod-1.11/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_11/LivingPatientSearchHandler1_11Test.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+
+package org.openmrs.module.webservices.rest.web.v1_0.search.openmrs1_11;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.module.webservices.rest.web.RestTestConstants1_11;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+
+
+import java.util.List;
+
+
+public class LivingPatientSearchHandler1_11Test extends  RestControllerTestUtils  {
+
+	private PatientService patientService;
+   
+	@Before
+	public void init() throws Exception {
+		patientService = Context.getPatientService();
+		executeDataSet(RestTestConstants1_11.LIVING_PATIENT_SEARCH_DATASET);
+	}
+	
+	protected String getURI() {
+		return "patient";
+	}
+	
+	@Test
+	public void SearchConfig_ShouldReturnNullPatientsIf_q_ParamIsNull() throws Exception {	
```

Comment:
Have you looked at our conventions for method names? https://wiki.openmrs.org/display/docs/Java+Conventions

Reply:
correctd naming